app/src/main/java/org/mozilla/fenix/home/HomeFragment.kt

```diff
@@ -964,8 +978,11 @@ class HomeFragment : Fragment() {
         )
     }
 
+    // TODO use [FenixTabCounterToolbarButton] instead of [TabCounter]: Link to issue
```

Comment:
Let's file and link to a new ticket here to use `TabCounterToolbarButton` in HomeFragment.

Reply:
https://github.com/mozilla-mobile/fenix/issues/16792